docs/en/administration/monitoring.md

```diff
@@ -5,24 +5,24 @@ sidebar_position: 3
 
 # Monitoring and Data Visualization
 
-As a distributed file system hosting massive data storage, we need to have the ability to visualize the status changes of the entire system in terms of capacity, files, CPU load, disk IO, cache, etc. JuiceFS provides real-time status data to the outside world through a Prometheus-based API. Just add it to your own Prometheus Server to scrape and store time series data, and then easily visualize and monitor the JucieFS file system with tools like Grafana.
+As a distributed file system hosting massive data storage, it is important for users to directly view the status changes of the entire system in terms of capacity, files, CPU load, disk IO, cache, etc. JuiceFS provides real-time status data externally through the Prometheus-oriented API to achieve the visualization of JuiceFS monitoring with ease, and you only need to expose it to your own Prometheus Server to visualize time series data with tools like Grafana.
```

Comment:
JuiceFS provides real-time metrics through Prometheus compatible API, add it to your own Prometheus scrape jobs, and then you can create alerts or visualizations using tools like Grafana.